x11-packages/qt5-qtkeychain/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/frankosterfeld/qtkeychain
+TERMUX_PKG_DESCRIPTION="Platform-independent Qt API for storing passwords securely."
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_MAINTAINER="Zhymabek Roman <robanokssamit@yandex.ru>"
+TERMUX_PKG_VERSION=0.14.2
+TERMUX_PKG_REVISION=1
```

Comment:
Since this is the initial packaging of qtkeychain, please remove the `TERMUX_PKG_REVISION=1` line.
This is only used when rebuilding a package without a version change.
```suggestion

```